components/script/dom/htmlaudioelement.rs

```diff
@@ -40,4 +46,19 @@ impl HTMLAudioElement {
             HTMLAudioElementBinding::Wrap,
         )
     }
+
+    pub fn Audio(
```

Comment:
Please add a link to https://html.spec.whatwg.org/multipage/media.html#dom-audio here.